[program:self-backup]
command=python3 /app/.vubuntu/assets/scripts/self-backup.py
exitcodes=0
startsecs=0
stderr_logfile=/app/.vubuntu/assets/logs/selfbackup.err.log
priority=6
